scripts/use-yarn-please.js

```diff
@@ -0,0 +1,58 @@
+const path = require('path');
+const { spawnSync } = require('child_process');
+const npmPath = process.env.npm_execpath;
+
+const Strings = {
+  useYarnInstead: `Looks like you are trying to run "npm install", this repository has migrated to use Yarn as its package manager for installations.
+  `,
+  yarnDetected: `Great! Looks like you already have yarn installed!
+  `,
+  installYarn: `You current do not have an installation of Yarn in your PATH. Be sure to install the latest stable version of Yarn here:
+
+Download an installer here: https://yarnpkg.com/en/docs/install
+
+Windows users can run the installer here:
+
+  https://yarnpkg.com/latest.msi
+
+Mac users can run a one-liner for the install:
+
+  curl -o- -L https://yarnpkg.com/install.sh | bash
+`,
+  gettingStartedWithYarn: `To install UI Fabric monorepo dependencies and establish links between projects, simply run:
+
+  yarn
+
+You can then build the packages
+
+  yarn build
+
+Or start developing innerloop against the UI Fabric demo with this command
+
+  yarn start
+
+To read more about all the commands that this mono-repo supports, go to this link:
+
+  https://github.com/OfficeDev/office-ui-fabric-react/wiki/Build-Commands
```

Comment:
Guessing you plan to update this shortly after the PR completes?

Reply:
I'll have a draft ready somewhere to review

Reply:
I think having a draft available to look at would be super helpful in getting a sense of what the "ergonomics" of the new system feel like and potentially improving things pre-checkin.